spyder/plugins/projects/plugin.py

```diff
@@ -904,3 +939,37 @@ def get_project_types(self):
             are project type classes.
         """
         return self._project_types
+
+    # --- Private API
+    # -------------------------------------------------------------------------
+    def _new_editor(self, text):
+        self.editor.new(text=text)
+
+    def _editor_open_files(self, _unused):

```

Comment:
Let's rename this `_unused` to `__unused`  to have the same signature as `_unset_path_in_editor` below.